src/js/components/Columns/stories/MainAside.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {
+  Box,
+  Columns,
+  Grommet,
+  Header,
+  Heading,
+  Main,
+  Menu,
+  Paragraph,
+} from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Next, Previous } from 'grommet-icons';
+
+const paragraphFiller = `
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+tempor incididunt ut labore et dolore magna aliqua.
+`;
+
+export const MainAside = () => (
+  <Grommet theme={grommet} full>
+    <Columns fill width="xlarge">
+      <Main gap="medium">
+        <Header border="bottom">
+          <Box direction="row" align="center">
+            <Heading size="small">Welcome</Heading>
+          </Box>
+          <Menu label="actions" />
+        </Header>
+        <Columns
+          columns={[
+            undefined,
```

Comment:
In this scenario, what if I wanted the columns to wrap one over the other. Is that supported?

Reply:
I guess I envisioned that typically when we have 2 column layouts and we hit a small breakpoint, we wrap the second column under the first one